### PR TITLE
Feature/ms 68 products list user

### PR DESCRIPTION
**Pull request by andrystylist**

### CODE REVIEW

**Review by jlvillegas04 — APPROVED** [review]
LGTM

File: src/components/Admin/Products/ProductForm.jsx [hunk 1]
```diff
@@ -1,10 +1,207 @@
...
+      .test(
+        'fileSize',
+        'File too large',
+        value => {
+          console.log('--- AC: ', {
+            value,
+            size: value?.size,
+            FILE_SIZE,
+            isValid: value?.size <= FILE_SIZE
+          })
```
Comment (ht1204) [inline src/components/Admin/Products/ProductForm.jsx:49]: is this log necessary yet?

File: src/components/Admin/Products/Products.jsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+          'warning'
+        )
+      })
+      .finally(() => {
+        setIsLoading(false)
+      })
+  }
+
+  const handleSave = ({ id, formData }) => {
+    console.log('-- AC: ', { id, formData })
```
Comment (ht1204) [inline src/components/Admin/Products/Products.jsx:70]: remove this log.

File: src/components/Admin/Products/UploadFile.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+
+function UploadFile ({ setFieldValue, ...rest }) {
+  const handleOnChange = (e) => {
+    // Object is possibly null error w/o check
```
Comment (ht1204) [inline src/components/Admin/Products/UploadFile.jsx:6]: this comment could not be necessary anymore.

File: src/components/Modal/ControlledModal.jsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  useEffect(() => {
+    const handleScape = (event) => {
+      if (event.key === 'Escape') {
+        handleClose()
+      }
+    }
+
+    document.addEventListener('keydown', handleScape)
+
+    // Función para deshabilitar el desplazamiento del documento cuando el modal está abierto
```
Comment (ht1204) [inline src/components/Modal/ControlledModal.jsx:29]: remove the comment

File: src/components/Modal/ControlledModal.jsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    const disableScroll = () => {
+      document.body.style.overflow = 'hidden'
+    }
+
+    // Función para habilitar el desplazamiento del documento cuando el modal se cierra
```
Comment (ht1204) [inline src/components/Modal/ControlledModal.jsx:34]: also this

File: src/components/Modal/ControlledModal.jsx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    const enableScroll = () => {
+      document.body.style.overflow = 'auto'
+    }
+
+    if (isOpen) { // Se abrió el modal
+      disableScroll()
+    } else { // Se cerró el modal
```
Comment (ht1204) [inline src/components/Modal/ControlledModal.jsx:41]: remove the conditional comments

File: src/services/ProductService.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+export const updateProduct = async (id, product) => {
+  const { data } = await http.patch(`/products/${id}`, product)
+  return data
+}
+
+export const searchProducts = (search) => {
+  const controller = new AbortController()
+
+  console.log('--- AC - searchProducts')
```
Comment (ht1204) [inline src/services/ProductService.js:16]: remove this log

**Review by ht1204 — COMMENTED** [review]
Please, address the comments for approving, thx.

**Review by ht1204 — APPROVED** [review]

**Pull request closed without merging**